src/lib/bootswatch/src/cerulean/bootstrap-markbind.css

```diff
@@ -0,0 +1,22 @@
+.alert-warning-darker {
```

Comment:
Thank you for the attempt, but we really shouldn't do this. It's a lot of effort for very little value.
My thoughts on styling: https://github.com/MarkBind/markbind/issues/386#issuecomment-408722347, https://github.com/MarkBind/markbind/issues/386#issuecomment-409066696, https://github.com/MarkBind/vue-strap/pull/124#issuecomment-578390642

Reply:
Noted 👍 

Hmm, shall we go with the barebones approach mentioned in https://github.com/MarkBind/markbind/issues/962 then?

It's plain, but still looks good imo, and dosen't require theme specific styling
I'll edit the main post with a new gif in a while